### ISSUE TITLE
Implement a menu system

### ISSUE THREAD

**Issue by fcheadle** [thesadrogue/TheSadRogue.Integration#15]
The example game, and perhaps the integration library itself, should contain a menu system that includes a main menu and a pause menu (these can be the same menu).

Does it make sense to include such a class in the integration lib?

**Comment by Thraka** [comment]
I'm not sure. It may make sense to add it to the SadConsole.Extended library.

Would it use anything from GoRogue?

**Comment by fcheadle** [comment]
No, if my Homicide Detective use case holds, this would really be done entirely sadconsole-side.

The example game should still include an example where I initialize it, to give players an idea to work from.
